testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java

```diff
@@ -131,8 +133,6 @@ protected void start() {
             shutdownThread = ProcessHelper.addShutdownHook(proc);
 
 
-            client = createClient();
```

Comment:
This should be replaced by {{ServerClientProvider.INSTANCE.setClient(createModelControllerClient())}}. Maybe ensure any previous client was closed as well. If a {{Server.stop()}} is executed followed by a {{Server.start()}} a new client needs to be created.